openwisp_users/tasks.py

```diff
@@ -49,17 +54,31 @@ def password_expiration_email():
         )
         .filter(query)
     )
+    email_counts = 0
```

Comment:
if we want to sleep at the 10th email we have to set this to 1

Reply:
The test here creates only 10 users, and the sleep is triggered after sending 10 emails. 

https://github.com/openwisp/openwisp-users/pull/368/files#diff-d5fb80ec22799c0be99cef82ca1ecf3dc5eab95fe1ef834d5168511e2be254ecR455-R468